client/src/utils/search.js

```diff
@@ -0,0 +1,25 @@
+import * as QueryString from 'query-string';
+
+/**
+ * Global function for performing monument searches
+ * The page will be redirected to the search results page with the proper query params
+ * @param changedParams The query params to override the current (if existent) query params with
+ * @param history       The react-router history prop must be passed in here so that we can use `.push`
+ */
+export default function search(changedParams, history) {
```

Comment:
Could we rename this to be `searchMonuments()`?

Reply:
I tried but there's already an action named searchMonuments causing some conflicts and I'm too lazy to think of another name